### ISSUE TITLE
False positives on survey sites and customer interaction platforms

### ISSUE THREAD

**Issue by ghost** [badmojr/1Hosts#874]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

**Lists in use:**
- [ ] mini
- [ ] Lite
- [x] Pro
- [x] Xtra
<br>

**Client:**

<br>

**Domains:**
```
opinions.research-interactive.com
surveymyopinion.researchnow.com
cint.com
liveperson.com
lpsnmedia.net
liveperson.net
asynchronousmessaging.dashboard.co.uk

```
<br>

**Details:**

Please could you whitelist:
opinions.research-interactive.com
surveymyopinion.researchnow.com
cint.com
liveperson.com
liveperson.net
asynchronousmessaging.dashboard.co.uk
lpsnmedia.net

(INCLUDING ALL SUB-DOMAINS)

In-relation to opinions.research-interactive.com, surveymyopinion.researchnow.com and cint.com these domains are only accessible by survey websites like Swagbucks etc, it is not featured anywhere else, therefore it is not a tracker, yes it does collect data but only when a user consents to this when signing up for the survey panel in the first place, hence why when you click on the domain itself it presents a white blank page (when not blocked of course), by this being blacklisted it is breaking the functionality and I have to change my DNS every time simply to access it, this is not a tracker or advertisement and the idea of DNS-based adblocking is not to break websites or/and portals.

In-relation to the domains liveperson.com, liveperson.net, asynchronousmessaging.dashboard.co.uk and lpsnmedia.net these are all customer interaction platforms and providers that are essential to allow companies to provide live chat and assistance to their customers like mobile banking etc. Please can you whitelist these too?

I really hope you can help, thank you.

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>

**Comment by badmojr** [comment]
Any way to reproduce this? Links perhaps?

**Comment by Eow2022yia** [comment]
> Any way to reproduce this? Links perhaps?

Well, the surveys are invite-link accessible only so it would be difficult to show you unless you are signed up etc.

However I can tell you the domains provided prior are the ones being blocked which is crashing my surveys and it gives me a dead response page.

Thank you :)

**Comment by Eow2022yia** [comment]
Unable to reproduce?

Try to click the links with the 1Hosts enabled and you can see the domains are blocked, try without and they are accessible.

How else can this be reproduced otherwise @badmojr ?

**Comment by Eow2022yia** [comment]
Please see the attached image on what the domain resolves to when loaded through the relevant survey panel:

Thanks.![image](https://user-images.githubusercontent.com/109627209/207976365-313a0048-8524-4271-93f0-20df5de195a5.jpeg)

**Comment by Eow2022yia** [comment]
Please also see the attached image on what the domain (surveymyopinion.researchnow.com) ![image](https://user-images.githubusercontent.com/109627209/207976587-fa5801b3-1c5e-4f14-bb19-520a50d631c7.jpeg)resolves to when loaded through the relevant survey panel:

Thanks.

**Comment by Eow2022yia** [comment]
Please also see the attached image on what the domain (surveymyopinion.researchnow.com) resolves to when loaded through the relevant survey panel:

Thanks.
![image](https://user-images.githubusercontent.com/109627209/207976844-41d15c2e-9592-40f3-a9c2-ff1ba394596a.jpeg)

**Comment by Eow2022yia** [comment]
Hi,

Also in-addition to the survey sites,

Here is how:

liveperson.com
adminlogin.liveperson.net
lp-07.asynchronousmessaging.dashboard.co.uk
asynchronousmessaging.dashboard.co.uk

Being blacklisted is effectively disabling messaging within banking applications.
![image](https://user-images.githubusercontent.com/109627209/210073510-22234d1a-250e-4db7-8f28-1544ce77e5cf.jpeg)https://user-images.githubusercontent.com/109627209/210073534-8570bf9b-74f3-4d11-9c5e-5bc299f2912c.MOV

**Comment by Eow2022yia** [comment]
![image](https://user-images.githubusercontent.com/109627209/216835623-0a01899f-6452-4697-9819-a7adcdd08842.jpeg)

**Comment by Eow2022yia** [comment]
Hi @badmojr would you be able to resolve this please?

**Comment by badmojr** [comment]
No. This is why I need access to the problematic site/app to further test as it's not possible to remove all subdomains because most of them are probably used for tracking purposes.

**Comment by Eow2022yia** [comment]
> No. This is why I need access to the problematic site/app to further test as it's not possible to remove all subdomains because most of them are probably used for tracking purposes.

Okay,
If you don't believe me,

Try out one of the banks affected by this issue (That use LivePerson for customer support) and try it for yourself.

**Comment by Eow2022yia** [comment]
@badmojr How are you going to verify this issue without accessing the platform yourself?

To do this you need to set-up your own bank account with one that uses LivePerson.
